content/en/logs/guide/logs-rbac.md

```diff
@@ -194,56 +120,21 @@ curl -X DELETE "https://app.datadoghq.com/api/v2/roles/<ROLE_ID>/permissions" -H
 {{% /tab %}}
 {{< /tabs >}}
 
-### Attach a user to a role
-
-Now that your roles are configured with their permissions, assign these roles to your users.
+### Assigning users to roles
 
 {{< tabs >}}
 {{% tab "UI" %}}
 
-In the [Team Section][1] of Datadog, go to the User tab. Pick a user and assign them either the `ACME Admin` or `ACME User` role, in addition to any roles they may already be assigned. More details on user management are available in the [Account Management][2] section.
+1. In the [Users Section][1] of Datadog, select a user and assign them either the `ACME Admin` or `ACME User` role.
 
-{{< img src="logs/guide/rbac/assign_user.png" alt="Delete invite on the grid view" style="width:60%;">}}
-{{< img src="logs/guide/rbac/assign_user2.png" alt="Delete invite on the grid view" style="width:60%;">}}
+{{< img src="logs/guide/rbac/assign_user2.png" alt="Delete invite on the grid view" style="width:90%;">}}
```

Comment:
```suggestion
{{< img src="logs/guide/rbac/assign_user2.png" alt="Assigning roles to a user in the edit user screen" style="width:90%;">}}
```

Reply:
Not sure if I'm missing something here, but this text seems to better match the image